SpatialGDK/Documentation/content/schema.md

```diff
@@ -0,0 +1,47 @@
+# Schema
+
+SpatialOS uses [schema (SpatialOS documentation)](https://docs.improbable.io/reference/13.6/shared/concepts/schema#schema) to generate APIs specific to the components in your game world. These APIs define how you can operate on entity components within SpatialOS. 
+
+## Generating schema
+
+To generate schema, select the **Schema** button in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through objects with replicated properties to generate the required schema files and then updates the [`SchemaDatabase`(Unreal GDK Glossary)]({{urlRoot}}/content/glossary#schemadatabase).
+
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)
+
+*Image: In the GDK toolbar in the Unreal Editor, select the* ***Schema*** *button.*
+
+As the GDK automatically generates all the schema you need, you do not have to write or edit schema manually when using the GDK.
+
+### When to generate schema
+
+You must generate schema:
+
+- When you add or change any replicated properties that you want to deploy to SpatialOS.
+- When you create a new map. This ensures that you generate schema for any unique objects on this map. 
```

Comment:
As snapshots don't contain any map specific information, this isn't required anymore.

Reply:
Nuked as part of the snapshot changes. Thanks :D

Reply:
Section now reads: 

```
You must generate schema when you add or change any replicated properties that you want to deploy to SpatialOS.
```

Is that correct?

Reply:
That is correct.